librz/include/rz_bin_dwarf.h

```diff
@@ -778,6 +778,7 @@ typedef struct {
 	 * that references this particular line information.
 	 */
 	HtUP /*<ut64, char *>*/ *line_info_offset_comp_dir;
+	size_t tot_dies;
```

Comment:
the code is ok, is just the name of this variable that is a bit weird. can this be renamed into something more clear? like `n_debug_infos`